contribute/design/proposal-cstor-orchestration.md

```diff
@@ -0,0 +1,57 @@
+# Managing cStor based OpenEBS Volumes. 
+
+## Background 
+
+
+## Design
+
+Provisioning of cStor based Volume involve two steps. 
+
+1. Creating cStor StoragePools:
+
+   * Admin will create SPC that contains information like:
+     - type=block
+     - name
+     - disks list (type of PDC or specific disks)
```

Comment:
What format are we using for disks? if we use generic names (e.g. /dev/sdb, /dev/sdc), we might end up with pool configurations that seem to be random (e.g. one nodea:/dev/sdb  == 1GB and nodeb:/dev/sdb == 2GB) which might not be expected directly from by the user. 

Would it be possible (and not considered an anti-pattern) to be very explicit about which disk(s) is/ are used? 

It is possible, to ID a disk uniquely across the cluster for example:

rs://host:nodea/device:/dev/disk-by-id/avcd-121312-hfjgdf

This is perhaps a little more work initially for the user but in the long run, will help us:

a) make sure we accurately create what is needed without ambiguity
b) id the device -- cluster-wide  in case of problems (and there will be problems) 
c) track which disks we are watching in Prometheus as the disk would be